app/constants/organisation-unit.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  OFFENDER_MANAGER: { name: 'offendermanager', ref: 'I', capacityView: 'workload_owner', displayText: 'Offender Manager' },
```

Comment:
Can we update the `name` property of offender manager to `offender-manager` so it matches the rest of the path style i.e. `caseload-capacity`?